libmambapy/src/main.cpp

```diff
@@ -807,6 +807,7 @@ PYBIND11_MODULE(bindings, m)
         .def_readwrite("version", &PackageInfo::version)
         .def_readwrite("build_string", &PackageInfo::build_string)
         .def_readwrite("build_number", &PackageInfo::build_number)
+        .def_readwrite("noarch", &PackageInfo::noarch)
```

Comment:
I don't know how important the `noarch` could be, but it seems to be missing in some of the `PackageInfo` methods compared to the other members. So maybe we should add it there as well? (for consistency and comprehensiveness).

Reply:
Noarch is definitely important. Where do you see it missing?